.npmrc

```diff
@@ -1 +1 @@
-@brown-ccv:registry=https://registry.npmjs.org/
\ No newline at end of file
+@brown-ccv:registry=https://npm.pkg.github.com
```

Comment:
I wasn't able to see `@brown-ccv/behavioral-task-trials` 2.0.0 at the npmjs registry, but I was able to get it via github with this change.  This seems relevant to an [open ticket](https://github.com/brown-ccv/behavioral-task-trials/issues/29) at behavioral-task-trials, so I left a comment there.